src/client.ts

```diff
@@ -5288,10 +5288,6 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
         ];
 
         if (this.supportsExperimentalThreads()) {
-            if (!timelineSet.canContain(event)) {
-                return undefined;
-            }
```

Comment:
In order to make sure we're not regressing https://github.com/matrix-org/matrix-js-sdk/pull/2444 and https://github.com/matrix-org/matrix-js-sdk/pull/2454 for sure, do we have a specific testing strategy in the Element app to reproduce previously?

Reply:
@t3chguy Any hints here?